src/frame.js

```diff
@@ -0,0 +1,54 @@
+class Frame {
+  constructor() {
+    this.scores = new Array;
+    this.updated = false;
+    this.final = 0;
+    this.added = false;
+  }
+
+  isSpare() {
+    if (this.scores.length == 2) {
+      return this.isSumTen();
+    } else {
+      return false;
+    }
+  }
+
+  isStrike() {
+    if (this.scores.length < 3) {
```

Comment:
and this? I don't fully understand how the length of scores array ties into a spare or strike being bowled.